<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tempInvalidPage</class>
 <widget class="QWidget" name="tempInvalidPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ВУТ</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbTempInvalidRegime">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTempInvalidRegime">
     <property name="text">
      <string>Режим ВУТ по умолчанию</string>
     </property>
     <property name="buddy">
      <cstring>cmbTempInvalidRegime</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblTempInvalidReason">
     <property name="text">
      <string>Причина ВУТ по умолчанию</string>
     </property>
     <property name="buddy">
      <cstring>cmbTempInvalidReason</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbTempInvalidReason">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblTempInvalidDoctype">
     <property name="text">
      <string>Документ ВУТ по умолчанию</string>
     </property>
     <property name="buddy">
      <cstring>cmbTempInvalidDoctype</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbTempInvalidDoctype">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblTempInvalidRequestFss">
     <property name="text">
      <string>Запрашивать СФР на наличие открытых ЭЛН при создании нового эпизода ВУТ</string>
     </property>
     <property name="buddy">
      <cstring></cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="chkTempInvalidRequestFss">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbTempInvalidDoctype</tabstop>
  <tabstop>cmbTempInvalidReason</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
